### PR TITLE
Address pylint 3.3 messages

### PR DESCRIPTION
**Pull request by jacobtylerwalls**

### CODE REVIEW

**Review by Pierre-Sassoulas — APPROVED** [review]

File: pylintrc [hunk 1]
```diff
@@ -307,6 +307,9 @@ exclude-protected=_asdict,_fields,_replace,_source,_make
 # Maximum number of arguments for function / method
 max-args=10
 
+# Maximum number of positional arguments for function / method.
+max-positional-arguments=8
+
```
[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply pylintrc:312]: And release it in 3.3.1 asap

[the remark this answers is not in the document]
Comment (jacobtylerwalls) [reply pylintrc:312]: I don't know, seems presumptuous to me. There are several options, fixing it, disabling it, adding inline disables for egregious cases. Doesn't seem any different to me than the usual pylint minor upgrade where each new check has to be evaluated. We called out in the release note that this new check has its own setting.

[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply pylintrc:312]: I agree with your point that the default should be less than max-args, but it seems like it's something that should be fixed in a major. If you already had ``max-args=12`` in the code imo it means you wanted to disable all existing too-many-arguments while still guaranteeing that it's not getting worse either (i.e. your acceptable number of positional args is going to be high too). It also pretty much means that your going to have new ``too-many-positional-arguments`` raised when upgrading which is bad for a minor. 
Also, ``too-many-positional-arguments`` is a refactor message: annoying to remove and probably not something that will be default when pylint become "sane" in 4.0.

[the remark this answers is not in the document]
Comment (jacobtylerwalls) [reply pylintrc:312]: > (i.e. your acceptable number of positional args is going to be high too)

I guess I don't think we can assume that.

I'm keeping an open mind, but I just think every pylint minor comes with estimating the cost of complying with every new message.

[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply pylintrc:312]: Fair enough.

**Pull request merged**